common/src/com/thoughtworks/go/domain/AgentInstance.java

```diff
@@ -168,7 +168,7 @@ public boolean canApprove() {
     }
 
     public void refresh(final AgentRuntimeStatus.ChangeListener changeListener) {
-        if (agentConfigStatus == AgentConfigStatus.Pending || agentConfigStatus == AgentConfigStatus.Disabled) {
```

Comment:
was there any specific reason why disabled agents were not refreshing the runtime status?